tendrl/gluster_integration/message/callback.py

```diff
@@ -489,7 +489,8 @@ def volume_delete(self, event):
             if fetched_volume.name == event['message']['name']:
                 fetched_volume.deleted = True
                 fetched_volume.deleted_at = time_utils.now()
-                fetched_volume.save()
+                sync_ttl = int(NS.config.data.get("sync_interval", 10))
+                fetched_volume.save(ttl=sync_ttl)
```

Comment:
this will help to remove volume quickly. Because as per new change volume TTL is based on no.of volumes, so TTL is very high.